pyrtl/inputoutput.py

```diff
@@ -669,9 +669,12 @@ def _to_verilog_combinational(self):
                 print('    assign %s = {%s};' % t, file=self.file)
             elif net.op == 'r':
                 pass  # do nothing for registers
-            elif net.op == 'm':
+            elif net.op == 'm':  # use always block and assign as Verilog register
+                print('    always @( posedge clk )', file=self.file)
+                print('    begin', file=self.file)
                 t = (self._varname(net.dests[0]), net.op_param[0], self._varname(net.args[0]))
-                print('        assign %s = mem_%s[%s];' % t, file=self.file)
+                print('        %s <= mem_%s[%s];' % t, file=self.file)
```

Comment:
@Gamrix i looked at the verilog output from this and i think you need to change how the memory output registers are declared in verilog.
![screenshot from 2016-07-01 16-07-06](https://cloud.githubusercontent.com/assets/468384/16536428/2c38e84e-3fa6-11e6-8e96-5c9a3446d5fd.png)
![screenshot from 2016-07-01 16-09-46](https://cloud.githubusercontent.com/assets/468384/16536436/42896ac4-3fa6-11e6-9354-d08030ec30dd.png)
The output wires need to be declared as regs, otherwise the verilog compiler will fail.